### ISSUE TITLE
About the regression targets for w and h

### ISSUE THREAD

**Issue by Sharpiless** [ultralytics/yolov5#3221]
## ❔Question
What I see used for W and H regression in both your yolov3 and v5 is:

> pwh = (ps[:, 2:4].sigmoid() * 2) ** 2 * anchors[i]

which is not the same with:

> torch.exp(x[..., 2:4]) * self.anchor

Does this mater？

**Comment by glenn-jocher** [comment]
@Sharpiless of course it matters. The YOLOv5 equations are a significant improvement in stability and useability over the YOLOv3 equations.
